main.cpp

```diff
@@ -265,6 +296,21 @@ struct IntType
 
     operator int() const { return *a; }
 
+    IntType& apply( std::function<IntType&( int& )> func )
+    {
+        if ( func != nullptr )
+            func( *a );
+        return *this;
+    }
+
+    using funcPtr = void(*)( int& );
```

Comment:
`using` directives follow the same rules as Type naming.
- [ ] Capitalize your alias.